inc/parser/xhtml.php

```diff
@@ -147,7 +147,7 @@ function document_end() {
                     }
 
                     // add footnote markup and close this footnote
-                    $this->doc .= $footnote;
+                    $this->doc .= '<span class="content">'.$footnote.'</span>';
```

Comment:
Could we ever have block elements inside a footnote? If yes, then this should rather be a `<div>`.